templates/artist/includes/artist_detail_updates.html

```diff
@@ -53,4 +49,8 @@
             <input type="submit" value="Submit" />
         </form>
     {% endif %}
+{% else %}
```

Comment:
I noticed this when I was working on the previous note, I think I found a fix for it